app/build.gradle

```diff
@@ -12,6 +12,7 @@ android {
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        vectorDrawables.useSupportLibrary = true
```

Comment:
what does this do? teach me

Reply:
I have no idea, the `app:srcCompat="@drawable/ic_add_circle_black_24dp"` part in `main_activity.xml` won't compile without it.